### ISSUE TITLE
Use readr::read_fwf for fixed-width formats?

### ISSUE THREAD

**Issue by leeper** [leeper/rio#18]
Sounds like Hadley's uncovered one of the reasons why `read.fwf` is so slow: https://twitter.com/hadleywickham/status/576116548886138881. Might be worth switching.

**Comment by hadley** [comment]
Any reason not to use `read_csv()` etc too?

**Comment by leeper** [comment]
@hadley It boils down to a choice between using `data.table::fread` versus **readr** as a whole. We debated this in #3.

I just ran some benchmarks on this, which make a strong case for switching to `read_fwf`, though: https://gist.github.com/leeper/19b32b14ac746734cc91#file-microbenchmarks-r

Relatedly, I actually think the `utils::read.fwf` issue merits a patch suggestion to R-Core because the number of times it reads from a file is absurd.
